.github/workflows/coverage.yml

```diff
@@ -0,0 +1,24 @@
+name: Coverage Reports
+
+on:
+  workflow_call:
+
+jobs:
+  coverage:
+    name: Coverage Reports
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        package: [caliper-core, caliper-fabric, generator-caliper]
```

Comment:
do we not want all the other packages ?

Reply:
This is because we're running unit tests only for these packages (since only these packages have unit tests). Most other packages simply run the linter when `npm test` is run, so we wouldn't have coverage information.